ui/cypress/e2e/explorer.test.ts

```diff
@@ -654,8 +654,23 @@ describe('DataExplorer', () => {
           })
         cy.getByTestID('_value-table-header').click()
         cy.get('.table-graph-cell__sort-asc').should('exist')
-        // TODO: complete testing when FE sorting functionality is sorted
-        // https://github.com/influxdata/influxdb/issues/16200
+        cy.getByTestID('_value-table-header').click()
+        cy.get('.table-graph-cell__sort-desc').should('exist')
```

Comment:
It's a bit of both. The only indication that a specific cell header isn't toggled by the front end sort is the classname associated with it, hence the check for it.

Reply:
i don't think it's necessary at all - it really feels like an implementation detail: we could change our underlying css so that we render columns sorted ascending with some kind squiggly underline and columns sorted descending with a dashed outline. making those changes would render this test invalid. 

either way, i think it's enough discussion on a test - i think this is a bit implementation heavy, but if you're comfortable with this, go for it